node/node.go

```diff
@@ -1164,31 +1284,34 @@ func (n *Node) OnStart() error {
 		return err
 	}
 
-	if n.config.FastSync.Version == "v0" {
-		// Start the real blockchain reactor separately since the switch uses the shim.
-		if err := n.bcReactor.Start(); err != nil {
-			return err
+	if n.config.Mode != cfg.ModeSeed {
+
+		if n.config.FastSync.Version == "v0" {
```

Comment:
is "v0" a constant somewhere that we could use?

Reply:
Nope, I don't think so. We could add one